modules/rosa-sdpolicy-platform.adoc

```diff
@@ -68,6 +68,24 @@ ifndef::openshift-rosa-hcp[]
 endif::openshift-rosa-hcp[]
 clusters at this time. However, custom labels are supported when creating new machine pools.
 
+[id="rosa-sdpolicy-node-lifecycle_{context}"]
+== Node lifecycle
+
+Worker node persistence is not guaranteed and may be replaced at any time as part of the normal operation and management of OpenShift.
+
+Several events affect the longevity of a {product-title} worker node:
+
+* Machine health checks are deployed and configured to ensure that a worker node with a `NotReady` status is replaced to ensure smooth operation of the cluster.
+* AWS EC2 instance retirement may occur at any time when AWS detects irreparable failure of the underlying hardware that hosts the instance.
+ifdef::openshift-rosa[]
+* During upgrades, a new node is provisioned to account for any loss of cluster resources during the upgrade process. Once an upgrade is complete, an older node is then removed from the cluster.
+endif::openshift-rosa[]
+ifdef::openshift-rosa-hcp[]
+* During upgrades, each node is fully replaced by an updated node.
+endif::openshift-rosa-hcp[]
+
+It is best practice to configure applications to be resilient of node replacements.
```

Comment:
Suggested adjustment: "For all containerized workloads running on a Kubernetes based system, it is best practice to configure applications to be resilient of node replacements."